cypress/e2e/view-configurations.spec.cy.ts

```diff
@@ -0,0 +1,90 @@
+import PageSteps from '../steps/page-steps';
+import {YasguiSteps} from '../steps/yasgui-steps';
+import {YasqeSteps} from '../steps/yasqe-steps';
+
+describe('View configurations', () => {
+
+   beforeEach(() => {
+      cy.intercept('/repositories/test-repo', {fixture: '/queries/default-query-response.json'}).as('getGuides');
+      PageSteps.visitConfigurationPage();
+      cy.clearLocalStorage('yasgui_config');
+      cy.clearLocalStorage('prefixes');
+   });
+
+   it('Should render all tabs by default', () => {
+      // GIVEN: "view-configurations" page is visited.
+
+      // THEN: all components of yasgui, yasr, yasqe have to be visible.
+      cy.get('.tabsList').should('be.visible');
```

Comment:
Move any direct DOM access in step files. In the tests there should be only step invocations and assertions.